RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java

```diff
@@ -100,18 +100,15 @@ public void changeControlModeWinch(ControlMode mode) {
      * Stops the elevator from moving. Used at the end of commands.
      */
     public void stopElevator() {
-    	// stops any current commands telling the elevator to move.
     	winch.disableControl();
     	System.out.println("Elevator has stopped.");
     }
     
     /**
      * TODO Make this actually work - It will drift and continually use a new point to hold the position
```

Comment:
I did want to delete this comment, a description of the method is present in the javadoc above.